src/test/java/jewellery/inventory/helper/OrganizationTestHelper.java

```diff
@@ -0,0 +1,64 @@
+package jewellery.inventory.helper;
+
+import java.util.List;
+import java.util.UUID;
+import jewellery.inventory.dto.request.OrganizationRequestDto;
+import jewellery.inventory.dto.response.ExecutorResponseDto;
+import jewellery.inventory.dto.response.OrganizationResponseDto;
+import jewellery.inventory.model.Organization;
+import jewellery.inventory.model.OrganizationPermission;
+import jewellery.inventory.model.User;
+import jewellery.inventory.model.UserInOrganization;
+
+public class OrganizationTestHelper {
+
+  public static Organization getTestOrganization() {
+    Organization organization = new Organization();
+    organization.setId(UUID.randomUUID());
+    organization.setName("Test Name");
+    organization.setAddress("Test Address");
+    organization.setNote("Test Note");
+    return organization;
+  }
+
+  public static Organization getTestOrganizationWithUserInOrganizations() {
+    Organization organization = new Organization();
+    organization.setId(UUID.randomUUID());
+    organization.setName("Test Name");
+    organization.setAddress("Test Address");
+    organization.setNote("Test Note");
```

Comment:
Pls extract these magic strings as constants if we are using them in multiple methods.